src/main/java/com/squareup/javapoet/TypeSpec.java

```diff
@@ -269,18 +280,21 @@ void emit(CodeWriter codeWriter, String enumName, Set<Modifier> implicitModifier
     }
   }
 
-  @Override public boolean equals(Object o) {
+  @Override
+  public boolean equals(Object o) {
     if (this == o) return true;
     if (o == null) return false;
     if (getClass() != o.getClass()) return false;
     return toString().equals(o.toString());
   }
 
-  @Override public int hashCode() {
+  @Override
```

Comment:
You are right, I can't reproduce it.
I guess it was my fault. When I have started working on this, I have used intellij's default formatter and I guess I have pressed the shortcuts for formatting code somehow (well I press this shortcut all the time automatically without noticing it).

Before committing and pushing I have applied 'Square' formatter and have assumed that it will reformat previous changes as well ...